src/styles/index.scss

```
@@ -1,3 +1,78 @@
-body {
+$days: (mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6);
+$cell-height: 100px;
+$cell-width: 100px;
```

Don't need to duplicate variables with the same values
```suggestion
$cell-size: 100px;
```